Things_You_Should_Know/things_you_should_know.md

```diff
@@ -0,0 +1,400 @@
+
+|**CODE**|**CONTAINER**|
+|:------:|:-----------:|
+| ⛔    | ✅          |
+
+The first time you use a custom Docker image, we will do a `docker pull` and pull all layers. These layers are stored on disk just as if you were using Docker on-premise. When we do a `docker pull`  after a app restart, we will only pull layers that have changed. If there have been no changes, we will simply use existing layers on disk.
+
+If you change workers for any reason, however, we will have to pull down all layers again. Your worker will change if you scale up/down or if you scale out to add additional workers. There are also rare cases where your worker may change without a scale operation, but these cases are uncommon.
+
+### Continuous Deployment requires that the image:tag combination remain the same
+
+|**CODE**|**CONTAINER**|
+|:------:|:-----------:|
+| ⛔    | ✅          |
+
+Docker containers in **App Service** are specified using an `image:tag` combination. Therefore, if you configure Continuous Deployment, you'll need to ensure that any updates to your Docker image don't change the tag.
+
+We've seen quite a few cases where customers will initially tag the image with a "v1" tag and enable CI. At some later time, they update the image and tag the new image with "v2" and then wonder why CI isn't working. Once you change the tag on an image, it's considered a different image from the initially configured image, and that will cause CI to not pick up the changes.
+
+> *Note:*
+> If you don't specify a tag, Docker will look for an image with a tag called "latest". Therefore, if your configuration doesn't explicitly use a tag and someone tags your image with anything other than "latest", it will break continuous deployment.
+
+## Configuration
+
+### You can enable and disable storage persistence with an app setting
+
+|**CODE**|**CONTAINER**|
+|:------:|:-----------:|
+| ⛔    | ✅          |
+
+You can use an app setting called `WEBSITES_ENABLE_APP_SERVICE_STORAGE` to control whether or not the `/home` directory of your app is mapped to **App Service** built in storage.
+
+Storage persistence can be a useful feature if you need to share information across instances (for scale up / scale out scenarios), or if you need information to persist across application restarts. However there are a few caveats to keep in mind:
+
+If '/home' directory is mapped to **App Service** built in storage, your application will restart when a storage fail-over occurs.
+
+> **Note:**
+> 
+> Permisions on the /home directory are set to `777`. You cannot change these permissions even if you attempt to do so from an initialization script or from SSH. 
+
+
+If you don't require file persistence, you can set this app setting to **false**. this will make your app resilient to storage failovers.
+
+The absence of this app setting will result in the setting being **"true"**. In other words, if this app setting does not exist for your app storage will be mounted.
```

Comment:
This feels like a roundabout way of saying that by default when you create a containerized app on App Service we automatically mount /home to App Service's remote storage.